src/components/header.vue

```diff
@@ -7,6 +7,9 @@
             </button>
             <div class="collapse navbar-collapse" id="navbarNav">
                 <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <router-link :to="{ name: 'get-tasks' }" class="nav-link">See Tasks</router-link>
```

Comment:
Ova ruta bi bilo najlogicnije da je home ruta, ja bih rekla. I naziv za rutu `get-tasks` nije dobar, to ce se videti gore u URL-u, logicnije bi bilo `tasks` samo